developer-docs/providers/implementers-guide.md

```diff
@@ -33,7 +33,7 @@ string  = (* any sequence of unicode code points that does not contain "::" *) ;
 qualified type name = [ parent type "$" ] type ;
 parent type         = type ;
 
-type       = package ":" [ module ":" ] type name ;
+type       = package ":" [ module ] ":" type name ;
```

Comment:
I think this is "mypkg::MyType" typically when the module is not available, not "mypkg:MyType".